packages/drivers/routerlicious-driver/src/documentStorageService.ts

```diff
@@ -105,14 +105,9 @@ export class DocumentStorageService implements IDocumentStorageService {
     }
 
     public async readBlob(blobId: string): Promise<ArrayBufferLike> {
-        const iso = IsoBuffer.from(await this.read(blobId), "base64");
-
-        // In a Node environment, IsoBuffer may be a Node.js Buffer.  Node.js will
-        // pool multiple small Buffer instances into a single ArrayBuffer, in which
-        // case we need to slice the appropriate span of bytes.
-        return iso.byteLength === iso.buffer.byteLength
-            ? iso.buffer
-            : iso.buffer.slice(iso.byteOffset, iso.byteOffset + iso.byteLength);
+        const value = await this.manager.getBlob(blobId);
```

Comment:
Might be worth adding in read() an assert that this.encoding === "base64". If it's not true, then read() would return garbage and we would likely throw down the line - it's much better to fail earlier (even better would be for read() to be correct, but given we are removing it soon, I'd not waste time).